### PR TITLE
build: Add additional includes

### PR DESCRIPTION
**Pull request by winksaville**
These are needed to compile on X86_64 Arch Linux:

 $ uname -a
 Linux 3900x 5.18.6-arch1-1 #1 SMP PREEMPT_DYNAMIC Wed, 22 Jun 2022 18:10:56 +0000 x86_64 GNU/Linux

### CODE REVIEW

File: src/instruction.cpp [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#include <bitset>
```
Comment (stuartmscott) [inline src/instruction.cpp]: Where does this file use bitset?

**Review by stuartmscott — COMMENTED** [review]
Thanks Wink!

Mostly looks good and makes sense, just wondering about the `bitset` include in `instruction.cpp`

File: src/instruction.cpp (hunk 1 above)
Comment (winksaville) [reply src/instruction.cpp]: Will check, maybe a mistake or a dependency, will check when I get home.

Comment (winksaville) [reply src/instruction.cpp]: Removed spurious `#include <bitset>` from instruction.cpp. Previously it was needed as originally I'd done the change on top of e14c76ff4518800d300dfacdfb7b7bbbe5c7e8d6 where it was needed. I then just rebased on top of master and now it's not needed :)

**Review by stuartmscott — APPROVED** [review]
LGTM!

**Comment by stuartmscott** [comment]
Congratulations on landing your first PR to Neuro! 🥳

**Pull request merged**